src/docker/start.sh

```diff
@@ -81,6 +81,21 @@ else
   print_header "No custom SSL certificate provided"
 fi
 
+cleanup() {
+  if [ -e config.sh ]; then
+    print_header "Cleanup. Removing Azure Pipelines agent..."
+
+    # If the agent has some running jobs, the configuration removal process will fail.
+    # So, give it some time to finish the job.
+    while true; do
+      ./config.sh remove --unattended --auth PAT --token "$$AZP_TOKEN" && break
+
+      echo "Retrying in 30 seconds..."
```

Comment:
Why 30 secs?

If there are no apparent reasons, I would propose 15 secs, as it can permit to shut down quicker containers and save money on the infra.